bigquery_etl/view/validate_views.py

```diff
@@ -22,8 +22,6 @@
     "sql/moz-fx-data-shared-prod/telemetry/clients_daily_scalar_aggregates_v1/view.sql",
     "sql/moz-fx-data-shared-prod/telemetry/clients_histogram_aggregates_v1/view.sql",
     "sql/moz-fx-data-shared-prod/telemetry/clients_probe_processes/view.sql",
-    # Java errors
-    "sql/moz-fx-data-shared-prod/firefox_accounts_derived/funnel_events_source/view.sql",
```

Comment:
Why is this needed? This PR seems to be passing `validate-views` test.

Reply:
Hah, it was removed. Yesterday it was giving me java errors.

Reply:
Hah, I suck at reading diffs.